### PR TITLE
One-click editor link for related papers emails

### PR DESCRIPTION
**Pull request by maxkfranz**
- Create empty doc when sending related papers email.
- Send editor link in template to related papers email.
- The email template needs to be updated to use the `editorUrl` variable.

Ref. : One-click editor link for related papers emails #925

### CODE REVIEW

File: src/server/routes/api/document/index.js [hunk 1]
```diff
@@ -1450,19 +1450,35 @@ const emailRelatedPaperAuthors = async doc => {
 
   const hasContact = paper => getContact(paper) != null;
 
+  const createEmptyDoc = async (authorName, authorEmail) => {
+    const provided = {
+      authorEmail,
```
Comment (jvwong) [inline src/server/routes/api/document/index.js:1455]: We need the `paperId` set here or else `postDoc` complains threre's not PubMed article found.

**Review by jvwong — CHANGES_REQUESTED** [review]

File: src/server/routes/api/document/index.js (hunk 1 above)
Comment (maxkfranz) [reply src/server/routes/api/document/index.js:1455]: I think hat `postDoc()`, if it doesn't already, should be allowed to fail in part or in whole.  There is no paper ID for this empty doc as per: 

- One-click editor link for related papers emails #925
- Post hoc specification of the paper in a factoid #901
- Post hoc titles etc #921
- Refactored RequestForm as a reusable component. #913

Comment (jvwong) [reply src/server/routes/api/document/index.js:1455]: Ah yes, empty doc my bad

**Review by jvwong — APPROVED** [review]
OK works for me

**Pull request merged**